torch_geometric/nn/models/explainer.py

```diff
@@ -388,3 +388,52 @@ def visualize_subgraph(self, node_idx: Optional[int], edge_index: Tensor,
         nx.draw_networkx_labels(G, pos, **label_kwargs)
 
         return ax, G
+
+    def visualize_feature_importance(
+        self,
+        node_feat_mask: Tensor,
```

Comment:
Pass in `Explanation` object instead, and look for `node_feat_mask` in that.

Reply:
Done! I had to pass in the `node_index` too to filter the `node_feat_mask`